js/views/buttonsView.js

```diff
@@ -83,12 +84,18 @@ export default class ButtonsView extends Backbone.View {
   }
 
   onFeedbackMessageChanged(model, changedAttribute) {
-    if (changedAttribute && this.model.get('_canShowFeedback')) {
-      // enable feedback button
-      a11y.toggleEnabled(this.$('.js-btn-feedback'), true);
-      return;
+    if (!this.model.get('_canShowFeedback')) return;
```

Comment:
Remove this line from `onFeedbackMessageChanged` and put it at the top of `enableFeedbackButton`